### PR TITLE
Net7 example

### PR DESCRIPTION
**Pull request by qmjava**

### CODE REVIEW

File: MoesifNet7Example/MoesifNet7Example/appsettings.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿{
+  "MoesifOptions": {
+    "ApplicationId": "eyJhcHAiOiI1MTk6MzQ1IiwidmVyIjoiMi4xIiwib3JnIjoiNDIwOjMyMCIsImlhdCI6MTY5NjExODQwMH0.Ved_dF4sHK-E2kTCHhK3pj4VNk6krgJC-w8gxvwxEQE",
```
Comment (praves77) [inline MoesifNet7Example/MoesifNet7Example/appsettings.json]: @qmjava we can not check-in any key/id/secrets.

Comment (qmjava) [reply MoesifNet7Example/MoesifNet7Example/appsettings.json]: yeah, forgot to save the change

**Comment by BillChirico** [comment]
So this includes all of the pages, is this required to be implemented? For example, we're writing our own front end.

**Comment by praves77** [comment]
> So this includes all of the pages, is this required to be implemented? For example, we're writing our own front end.

@BillChirico no, this is not required to be implemented. We are cleaning it up to have a bare minimum .Net7 example.

**Comment by praves77** [comment]
Closing it because we are preparing a streamlined Moesif SDK integration example for .NET7, mirroring our .NET6 example. We will share this example very soon. BTW, there is no update pending for our SDK.

**Comment by BillChirico** [comment]
> Closing it because we are preparing a streamlined Moesif SDK integration example for .NET7, mirroring our .NET6 example. We will share this example very soon. BTW, there is no update pending for our SDK.

So the SDK won't be updated for .NET 7? I got it working with .NET 7 already, but wondering what changes to expect if any.

**Comment by praves77** [comment]
@BillChirico Yes, our SDK is up to date with respect to the feature parity and supports various .NET versions for backward compatibility. There is no update pending specific to .Net7 or previous versions. 
Since, you got it working with .NET 7 that means you are all set to send events/logs to the moesif.

Hope that helps. Please let the support team know via our app if you have more questions or needs to walk-through any features/demo.

**Comment by BillChirico** [comment]
@praves77 Thank you very much! Will keep you guys posted if we run into any issues.

**Pull request closed without merging**